items.json

```diff
@@ -15252,9 +15252,9 @@
     "name": "Heavy Flanged Mace",
     "culture": "Khuzait",
     "type": "OneHandedWeapon",
-    "price": 198,
-    "weight": 2.29,
-    "tier": 0.7856775,
+    "price": 2020,
+    "weight": 1.97,
+    "tier": 4.66271448,
     "requirement": 0,
     "flags": [],
     "weapons": [

```

Comment:
is this begginner tier on purpose? If the answer is yes i'm fine with it

Reply:
Made the tier a little higher